docs/Documentation/Development/Containers/apptainer.md

```diff
@@ -282,22 +282,248 @@ Process 4 exiting
 
 ```
 
+## Utilizing GPU resources with Apptainer images
```

Comment:
@mselensky I integrated the GPU content into this page and replaced $LOCAL_SCRATCH with /scratch/$USER. Can you review this section and verify the content is organized how you'd like?

Reply:
@yandthj I made a few minor changes to update the latest version of CUDA (and associated GPU drivers) we now have on Kestrel/Swift, otherwise this looks good to me!